ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -4906,7 +4906,6 @@ periodics:
   decorate: true
   interval: 8h
   labels:
-    ci.openshift.io/release-type: informing
```

Comment:
I think we still want this.  OVN is still critical on aws for 4.4 so we need to keep this job on our informing list.